src/public/media.ts

```diff
@@ -277,6 +292,25 @@ export namespace media {
     enableFilter?: boolean;
   }
 
+  /**
+   * @private
```

Comment:
Same question; why is this being marked as private?

Reply:
Launching lens camera in only Video mode is not supported as of now. Once the lens enables this support we will expose it to developers for use. 

For now, VideoProps is required for VideoAndImageProps (mediatype = 3) and hence kept it private to hide it from public docs.